contracts/amm/WethHelper.sol

```diff
@@ -0,0 +1,111 @@
+pragma solidity 0.6.12;
+
+import "@openzeppelin/contracts-ethereum-package/contracts/token/ERC20/IERC20.sol";
+import "@openzeppelin/contracts-ethereum-package/contracts/token/ERC20/SafeERC20.sol";
+
+import "../market/IMarket.sol";
+
+interface IWETH {
+    function deposit() external payable;
+    function transfer(address to, uint value) external returns (bool);
+    function withdraw(uint) external;
+}
+
+interface IMinterAmm {
+    function bTokenBuy(
+        uint256 marketIndex,
+        uint256 bTokenAmount,
+        uint256 collateralMaximum
+    ) external returns (uint256);
+
+    function bTokenSell(
+        uint256 marketIndex,
+        uint256 bTokenAmount,
+        uint256 collateralMinimum
+    ) external returns (uint256);
+
+    function provideCapital(uint256 collateralAmount, uint256 lpTokenMinimum) external;
+
+    function getMarket(uint256 marketIndex) external view returns (IMarket);
+
+    function lpToken() external view returns (ISimpleToken);
+}
+
+contract WethHelper {
+    using SafeERC20 for IERC20;
+
+    address internal constant WETH = 0xC02aaA39b223FE8D0A0e5C4F27eAD9083C756Cc2;
+
+    constructor() public {
+
+    }
+
+    function provideCapital(address poolAddress, uint256 collateralAmount, uint256 lpTokenMinimum)
+        external payable
+    {
+      require(msg.value > 0, "No value passed");
```

Comment:
should we also check that `msg.value == `collateralAmount`, since if `msg.value < collateralAmount` it will fail, and if `msg.value > collateralAmount` then there will be residual ETH left over on the WETHHelper.